packages/@o3r/styling/builders/style-extractor/index.ts

```diff
@@ -40,9 +70,9 @@ export default createBuilder<StyleExtractorBuilderSchema>(async (options, contex
       // Check duplicate CSS variable
       cssVarList
         .filter((cssVar) => !!acc.variables[cssVar.name])
-        .filter((cssVar) => !initialPreviousMetadata.variables[cssVar.name] && (acc.variables[cssVar.name] as CssVariable).defaultValue !== cssVar.defaultValue)
+        .filter((cssVar) => !initialPreviousMetadata.variables[cssVar.name] && acc.variables[cssVar.name].defaultValue !== cssVar.defaultValue)
```

Comment:
```suggestion
        .filter((cssVar) => initialPreviousMetadata.variables[cssVar.name]!.defaultValue !== cssVar.defaultValue)
```

Reply:
As discussed it's not doing exactly the same thing